independent-publisher-connectors/Star Wars/apiDefinition.swagger.json

```diff
@@ -0,0 +1,878 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Star Wars",
+    "description": "A long time ago in a galaxy far, far away....",

```

Comment:
@paulculmsee It looks good as a caption for your connector, but we need a description for your connector which can be understandable by everyone. Please provide a detailed description that should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
That one I thought might get through on pop culture alone. But corrected with API description